Refit.Tests/RequestBuilder.cs

```diff
@@ -425,6 +426,9 @@ public interface IDummyHttpApi
 
         [Get("/foo/bar/{id}")]
         Task<string> FetchSomeStuffWithQueryFormat([Query(Format= "0.0")] int id);
+
+        [Get("/query")]
+        Task QueryWithEnumerable(IEnumerable<int> numbers);
```

Comment:
What about arrays? The behavior should be the same, right?

Reply:
I can add another specific test, but yes, it should. The changes underneath in there work on `object` as the `value`

Reply:
New test added